spring-integration-core/src/test/java/org/springframework/integration/configuration/EnableIntegrationTests.java

```diff
@@ -706,7 +707,8 @@ public void testSourcePollingChannelAdapterOutputChannelLateBinding() {
 
 	@Test
 	public void testIntegrationEvaluationContextCustomization() {
-		EvaluationContext evaluationContext = this.context.getBean(EvaluationContext.class);
+		EvaluationContext evaluationContext = this.context
+				.getBean(IntegrationContextUtils.INTEGRATION_EVALUATION_CONTEXT_BEAN_NAME, EvaluationContext.class);
```

Comment:
Looks like the `this.context.getBean(StandardEvaluationContext.class);`
And this way we really ensure that `simple` variant is really not a `StandardEvaluationContext` impl.

Or do you have something else in mind on the matter ?

Reply:
No; that's good; I was just fixing the test because there are now 2 ECs.